enhancements/ingress/admin-configured-ca-bundle.md

```diff
@@ -0,0 +1,240 @@
+---
+title: admin-configured-ca-bundle
+authors:
+  - "@bhb"
+reviewers:
+  - "@candita" ## reviewer for cluster-ingress-operator component
+  - "@Miciah" ## reviewer for cluster-ingress-operator component
+  - "@tgeer" ## reviewer for CFE
+approvers:
+  - "@candita" ## approval from cluster-ingress-operator component
+  - "@Miciah" ## approval from cluster-ingress-operator component
+  - "@jerpeter1" ## approval from CFE
+api-approvers:
+  - None
+creation-date: 2023-11-02
+last-updated: 2023-12-01
+tracking-link:
+  - https://issues.redhat.com/browse/RFE-2182
+  - https://issues.redhat.com/browse/OCPSTRAT-431
+  - https://issues.redhat.com/browse/NE-761
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Support for admin configured CA trust bundle in Ingress Operator
+
+## Summary
+
+This enhancement describes the proposal to allow an administrator of OpenShift to configure
+trusted Certificate Authorities (CA) to verify the services' certificate by OpenShift Router
+for the re-encrypted traffic.
+
+## Motivation
+
+When an OpenShift administrator creates a route with the re-encrypt termination type, they
+have to also provide a CA certificate for that route if it uses a custom CA instead of the
+default OpenShift CA for certificate validation. In addition, when the same CA is used by the
+administrator for signing several services' certificates it is time-consuming and error-prone
+to configure the same CA certificate in every created route object.
+Likewise, we'd like to improve the process for updating a CA certificate when expired,
+compromised or for any other reason in each route object. Instead, this document proposes a
+simpler alternative: a feature in which an administrator could define a CA bundle for verifying
+certificates from a ConfigMap consisting of the list of CA certificates, associated OpenShift
+Ingress Controller, and trusted OpenShift CA.
+
+### User Stories
+- As an OpenShift cluster administrator, I want to configure admin CA certificate bundle which
+  should be used for verifying service's certificate by the OpenShift router, instead of
+  having to configure the same as destinationCACertificate in each of the created route object.
+- As an OpenShift developer, I want to create a CA trust bundle for Ingress operator usage,
+  which includes the OpenShift service CA and the admin provided CA bundle as the default CA
+  for verifying the service's certificate when a destination CA certificate is not configured
+  for a route.
+- As an OpenShift developer, I want to update the OpenShift router certificate cache whenever
+  the OpenShift service CA certificate or the admin configured CA certificate bundle is updated
+  or deleted.
```

Comment:
Add one more user story so that QE is aware this is still possible, and it is still tested:
As a developer, I want to be able to continue to override the default CA's by router annotation.

Reply:
I would add an explicit story to ensure the ability to override an admin default is tested by QE.